symengine/ntheory.cpp

```diff
@@ -1498,4 +1498,15 @@ int mobius(const Integer &a)
     }
 }
 
+int mertens(const int long a)
```

Comment:
This can just be `unsigned long` and then you don't have to check for negative values